s30384-pj-AleksanderKoss/zadania1-30.09.23.txt

```diff
@@ -133,9 +158,11 @@ mkdir katalog2 katalog3
 cd ~
 tree
 
-- nauczyłem się jak używać komendy "cd" do poruszania się między katalogami oraz, że komenda "mkdir" tworzy nowy katalog w katalogu w którym aktuanie się znajduję. 
-- Następnie użyłem komendy  cd ~, aby przenieść się do katalogu domowego, a następnie komendy tree, aby wyświetlić drzewo katalogów w "home", 
-  żeby sprawdzić czy dobrze wykonałem strukturę katalogów
+- nauczyłem się jak używać komendy "cd" do poruszania się między katalogami oraz, 
+że komenda "mkdir" tworzy nowy katalog w katalogu w którym aktuanie się znajduję. 
```

Comment:
To działa tak samo jak `ls` bez parametrów. Też działa w aktualnym katalogu.
A kto broni wpisać coś w tym stylu `mkdir -p ../../../tmp/ala/ma/kota` ?